cmake/projects/cvmatio/hunter.cmake

```diff
@@ -0,0 +1,40 @@
+# This is a header-like file, so include guards needed
+if(DEFINED HUNTER_CMAKE_PROJECTS_CVMATIO_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_CVMATIO_HUNTER_CMAKE_ 1)
+endif()
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_add_version_start)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_add_package)
+
+hunter_add_package(OpenCV)
```

Comment:
This line not needed. This dependency can be managed in `cvmatio` itself. Let's add OpenCV to Hunter first and I will send you pull request to `cvmatio`.